ui/app/ducks/gas/gas.duck.js

```diff
@@ -111,26 +115,36 @@ async function basicGasPriceQuery() {
 
 export function fetchBasicGasEstimates() {
   return async (dispatch, getState) => {
+    const isMainnet = getIsMainnet(getState());
+    const currentNetworkId = getCurrentNetworkId(getState());
```

Comment:
The network ID is deprecated; we should only use the chain ID to unique identify networks. The network ID doesn't work as a unique identifier because different networks can share the same network ID. The chain ID on the other hand is always different per network.